src/components/services-view/services-view.controller.ts

```diff
@@ -81,7 +81,7 @@ export class ServicesViewController implements angular.IController {
       this.ctrl.currentSubFilter = null;
       this.ctrl.expandSubCatRow = ServicesViewController.NO_SUBCATEGORY_ROW;
     } else {
-      this.ctrl.filterByCategory(this.ctrl.currentFilter, subCategory, false);
+      this.filterByCategory(this.ctrl.currentFilter, subCategory, false);
```

Comment:
'this.ctrl.filterByCategory' is in my code base, seems to work, and I don't get a runtime exception?